src/main/java/io/supertokens/storageLayer/StorageLayer.java

```diff
@@ -175,7 +182,7 @@ private static StorageLayer getInstance(TenantIdentifier tenantIdentifier, Main
     public static void initPrimary(Main main, String pluginFolderPath, JsonObject configJson)
             throws MalformedURLException, InvalidConfigException {
         main.getResourceDistributor().setResource(new TenantIdentifier(null, null, null), RESOURCE_KEY,
-                new StorageLayer(main, pluginFolderPath, configJson));
+                new StorageLayer(main, pluginFolderPath, configJson, TenantIdentifier.BASE_TENANT));
     }
```

Comment:
add test that if tenant is not the base tenant, the logging doesn't happen to the console